governance/polls/Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022.md

```diff
@@ -0,0 +1,59 @@
+
+---
+title: Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022
+summary: Signal your support or opposition to the following liquidation parameters, designed to offboard UNIV2UNIETH-A
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-30T16:00:00
+end_date: 2022-06-02T16:00:00
+---
+# Poll: Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 30 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNIV2UNIETH-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNIV2UNIETH-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```

Comment:
Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

Reply:
Fixed on all!